packages/redux/src/checkout/draftOrders/reducer.ts

```diff
@@ -0,0 +1,176 @@
+import * as actionDraftOrdersTypes from './types/draftOrdersAction.types.js';
+import * as actionTypes from './actionTypes.js';
+import { type AnyAction, combineReducers, type Reducer } from 'redux';
+
+import createReducer, {
+  createReducerWithResult,
+} from '../../helpers/reducerFactory.js';
+import type { DraftOrdersState } from './types/index.js';
+import type { StoreState } from '../../types/storeState.types.js';
+
+export const INITIAL_STATE: DraftOrdersState = {
+  draftOrder: {},
+  allDraftOrders: {},
+  draftOrderCreations: {},
+  updateDraftOrder: {},
+  removeDraftOrder: {},
```

Comment:
Hello! Following this approach is not obvious from the code what the `isLoading` and `error` values mean. I think having a `createDraftOrder` entry containing the `isLoading` and `error` variables are preferable.
Regarding condensing the remove and update actions, this might be a problem since the user will not know what the error was for (was it for the update or the remove action?) If that is not necessary to distinguish, then yes it can be condensed, otherwise I think this could be split. There are some reducers that do that yes but I am not convinced that is the best approach here to be honest. Anyway, the more important part are the selectors api, if they are ok, we can revisit this later.